gTea/app/src/main/java/com/nechaieva/gtea/NavigationFragment.java

```diff
@@ -30,5 +34,19 @@ public void onClick(View view) {
                         .navigate(R.id.action_NavFragment_to_SecondFragment);
             }
         });
+
+        RecyclerView recyclerView = (RecyclerView) view.findViewById(R.id.menu_items);
+
+        // use this setting to improve performance if you know that changes
+        // in content do not change the layout size of the RecyclerView
+        recyclerView.setHasFixedSize(true);
+
+        // use a linear layout manager
```

Comment:
nit (i.e. short for nit pick): only use comments if it is necessary to add additional information that is not directly clear from the code. Here I would opt for removing the comment because it is clear that we use a LinearLayoutManager

Reply:
This and some of the following comments are on the outdated commits. The obvious comments were deleted later on. Possibly, however, you mean that they are unnecessary in the history at all? I can alter it, but it's a somewhat in-obvious operation.

Reply:
No keep the history, I was not aware that they are deleted later.